.github/workflows/release.yml

```diff
@@ -0,0 +1,60 @@
+name: release
+
+on:
+  push:
+    tags:
+      - v*.*.*
+
+env:
+  DOCKER_BUILDKIT: 1
+  DOCKER_REGISTRY: docker.elastic.co
+  DOCKER_IMAGE_NAME: observability/apm-lambda-extension
+
+permissions:
+  contents: write
+  pull-requests: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    outputs:
+      os: ${{ steps.artifacts.outputs.os }}
+      arch: ${{ steps.artifacts.outputs.arch }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+      - uses: docker/setup-qemu-action@v2
+        with:
+          platforms: linux/arm64, linux/amd64
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - uses: elastic/apm-pipeline-library/.github/actions/docker-login@current
+        with:
+          registry: docker.elastic.co
+          secret: secret/observability-team/ci/docker-registry/prod
+          url: ${{ secrets.VAULT_ADDR }}
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+      - uses: hashicorp/vault-action@v2
```

Comment:
what do you think to use the same version we use in the apm-pipeline-library? `hashicorp/vault-action@v2.4.2` see https://github.com/elastic/apm-pipeline-library/blob/main/.github/actions/setup-vault-cli/action.yml#LL20C15-L20C44

Reply:
https://github.com/elastic/apm-aws-lambda/pull/362/commits/b23272aaa3dcea334526b36ca458d0406d31f04b